### ISSUE TITLE
Old Varrock guards

### ISSUE THREAD

**Issue by kingOfvarrock** [runelite/runelite#4024]
I would love to be able to swap out the current ugly, too-many-polygons RS3 type guards roaming around Varrock with the better suited old model which was similar to Falador guards.

Is it possible to add a feature to use the old guards just like we can change the game-frame?

**Comment by MESLewis** [comment]
Depends on #7954

**Comment by Hydrox6** [comment]
Duplicate of #3261
